tests/rptest/services/redpanda.py

```diff
@@ -83,11 +84,17 @@
 
 
 class BadLogLines(Exception):
-    def __init__(self, nodes):
-        self.nodes = nodes
+    def __init__(self, node_to_lines):
+        self.node_to_lines = node_to_lines
 
     def __str__(self):
-        return f"<BadLogLines nodes={','.join([n.account.hostname for n in self.nodes])}>"
+        # Pick the first line from the first node as an example, and include it
+        # in the string output so that for single line failures, it isn't necessary
+        # for folks to search back in the log to find the culprit.
+        example_lines = next(iter(self.node_to_lines.items()))[1]
+        example = next(iter(example_lines))
+
+        return f"<BadLogLines nodes={','.join([f'{i[0].account.hostname}({len(i[1])})' for i in self.node_to_lines.items()])} example=\"{example}\">"
```

Comment:
Line length?

Reply:
Interesting, I don't usually do any formatting & leave it to my auto-run of the linter.  In this case apparently a line this long was within the linter rules, so it didn't break it up, presumably because it doesn't know how to handle the long `{}` expressions inside a string.  Broke this up by hand.